src/Microsoft.Identity.Core/Helpers/Base64UrlEncoder.cs

```diff
@@ -29,7 +29,7 @@
 using System.Globalization;
 using System.Text;
 
-namespace Microsoft.IdentityModel.Clients.ActiveDirectory.Internal.Helpers
+namespace Microsoft.Identity.Core.Helpers
```

Comment:
>Microsoft.Identity.Core.Helpers [](start = 10, length = 31)

I assume the namespace should still be internal ?

Reply:
not sure , core project is kind of inherently internal